docs/reference_guide.md

```diff
@@ -1829,6 +1831,42 @@ NetworkManager:1155 /var/lib/sss/mc/passwd (deleted)
 
 ```
 
+## 18. `rawtracepoint`: Static Tracing, Kernel-Level
+
+Syntax: `rawtracepoint:name` or `rt:name`
+
+These use tracepoints (a Linux kernel capability).
+
+```
+# bpftrace -e 'rawtracepoint:block_rq_insert { printf("block I/O created by %d\n", tid); }'
+Attaching 1 probe...
+block I/O created by 189126
+[...]
+```
+
+## 19. `rawtracepoint`: Static Tracing, Kernel-Level Arguments
+
+Example:
+
+```
+# bpftrace -e 'rawtracepoint:block_rq_insert { printf("%llx %llx\n", arg0, arg1); }'
+Attaching 1 probe...
+ffff88810977d6f8 ffff8881097e8e80
+[...]
+```
+
+The available members for each tracepoint can be can be found in the relative path of the kernel source code include/trace/events/. For example:
+```
+include/trace/events/block.h
+
+DEFINE_EVENT(block_rq, block_rq_insert,
+
+	TP_PROTO(struct request_queue *q, struct request *rq),
+
+	TP_ARGS(q, rq)
+);
+```
```

Comment:
Mention here that the individual args are accessed by their order via `argN` builtins and that each arg is 64-bit.